### PR TITLE
Doc: E.g. Git text confusing. PCRE->PCRE2

### PR DESCRIPTION
**Pull request by PallHaraldsson**

### CODE REVIEW

**Comment by PallHaraldsson** [comment]
"Git linkable"?! Just say "library" or as I did, or "dynamically"?

PCRE[2] Wikipedia article "says: 8.39 / June 14, 2016; 1 day ago"

Should text say "PCRE2"? May not be important [Ubuntu PPA says pcre3..]

README has e.g. (partly) lower case "FemtoLisp", should news LICENSE follow case for that and other, e.g. "libgit2"?

"utf8proc]** (>= 2.0)" -> 2.0.1? 2.0 has security fix (but not 2.0.1?), so backport fix (and text)?

**Comment by tkelman** [comment]
> Should text say "PCRE2"? May not be important [Ubuntu PPA says pcre3..]

Sure. pcre3 is a strange version number that I don't think is meaningful.

When you want to skip CI, you should spell it `[ci skip]` at the end of your commit message (doesn't have to be the first line). Make sure to run `make check-whitespace` locally first, if you can.

**Comment by PallHaraldsson** [comment]
There's a "typo" in the second commit: News this"->"Should this".

Please squash or git amend first if you do a pull, I've yet figured it out (seems I need to connect from git CLI, not GitHub).

File: README.md [hunk 1]
```diff
@@ -309,7 +309,7 @@ For a longer overview of Julia's dependencies, see these [slides](https://github
 [AMOS]:         http://netlib.org/amos
 [ARPACK]:       http://forge.scilab.org/index.php/p/arpack-ng
 [FFTW]:         http://www.fftw.org
-[PCRE]:         http://www.pcre.org
+[PCRE2]:     http://www.pcre.org
```
Comment (Keno) [inline README.md]: Alignment here. Also needs a rebase.

**Comment by PallHaraldsson** [comment]
@keno I see in here that PCRE2 got misaligned, but GitHub edit here:

https://github.com/PallHaraldsson/julia/edit/patch-11/README.md?pr=/JuliaLang/julia/pull/17430

doesn't show that.. so harder to fix.. I'm doing a git clone as we speak (up to 70 MB downloaded right now for a trivial doc change..).

I'm not even sure the rebase (or alignment) matters with "All checks have passed"?

**Comment by PallHaraldsson** [comment]
DO NOT MERGE, but

if anyone can help (e.g. @tkelman as I see his comment here..:

Merge pull request #17371 from JuliaLang/cb/dbldashparsing
tkelman committed with PallHaraldsson on 12 Jul

Note, I do not remember having had anything to do with that..)

then at least "Files changed" seems ok now. Every time I rebase, I end up in some weird stuff and do not squash, just make commit list longer..

See e.g. last two commits (and first, that is mine, just something completely unrealated.

Should I not start with:

git rebase -i HEAD~25

I get an even longer list than 25 commits, but I think all those 25 on GitHub are included.

**Comment by tkelman** [comment]
better to start this over as a single commit, given how small the change is

**Pull request closed without merging**